src/graph/index.d.ts

```diff
@@ -17,7 +17,7 @@ export type QueryArgsNoDeps<
   A extends IOTSParams,
   P
   > = {
-    id: string,
+    id?: string,
```

Comment:
Why optional?

Reply:
Because it is still used as the name for the caches used in each node, and this can be helpful when debugging with  `DEBUG=avenger*`: the logs are colored and organized by name of the cache, and the default name if not passed is just "anonymous".

We could make this more explicit and call it "cacheName" maybe?

Reply:
or `debugName`... up to you